src/platform/forms/tests/save-in-progress/RoutedSavableApp.unit.spec.jsx

```diff
@@ -432,6 +435,6 @@ describe('Schemaform <RoutedSavableApp>', () => {
 
     expect(fetchInProgressForm.called).to.be.false;
     expect(router.replace.calledWith('/first-in-form-page')).to.be.true;
-    __BUILDTYPE__ = buildType;
+    environment.BUILDTYPE = buildType;
```

Comment:
I didn't want to remove it but I don't think `BUILDTYPE` here really matters, because unit tests run under `vagovdev` and the corresponding component only has unique behavior during `localhost` builds.